src/Sweep.cpp

```diff
@@ -1330,7 +1330,7 @@ void DigitalContactTracingSweep(double t)
 						if ((Hosts[contact].dct_test_time == ts) && (Hosts[contact].index_case_dct == 0))
 						{
 							//if host is positive
-							if ((abs(Hosts[contact].inf) == 2) || (Hosts[contact].inf == -1)) //either asymptomatic infectious, symptomatic infectious or presymptomatic infectious
+							if ((abs((int) Hosts[contact].inf) == 2) || (Hosts[contact].inf == InfStat::InfectiousAlmostSymptomatic)) //either asymptomatic infectious, symptomatic infectious or presymptomatic infectious
```

Comment:
Replace the "2" with InfStat::InfectiousAsymptomaticNotCase

Reply:
I wonder about expanding the abs(...) == 2, to something explicitly stating the two cases - (InfStat_InfectiousAsymptomaticNotCase || InfStat_Case) - I think that would be clearer, and probably no worse than performing abs()